baselines/ts5.5/audioworklet.generated.d.ts

```diff
@@ -934,7 +934,7 @@ interface TextEncoder extends TextEncoderCommon {
      *
      * [MDN Reference](https://developer.mozilla.org/docs/Web/API/TextEncoder/encode)
      */
-    encode(input?: string): Uint8Array;
+    encode(input?: string): Uint8Array<ArrayBuffer>;
```

Comment:
This is wrong for TS 5.5. The minimum should be TS 5.7. How do I deal with this?